templates/contact-us.html

```diff
@@ -21,23 +21,122 @@
+        <p class="u-sv1">Publishing or distributing Ubuntu in any form other than our standard install media and packages
+          requires permission. To license the Ubuntu trademark or any other rights, please <a
+            href="https://www.ubuntu.com/legal/terms-and-policies/contact-us">submit an enquiry</a>.</p>
         <p class="p-heading--three">Press and analyst enquiries</p>
         <p class="u-sv1">Please contact <a href="mailto:pr@canonical.com">pr@canonical.com</a>.</p>
         <p class="p-heading--three">Data Privacy enquiries</p>
-        <p class="u-sv1">For information about data privacy, please read our <a href="https://www.ubuntu.com/legal/data-privacy">Privacy Policy</a> or <a href="https://ubuntu.com/legal/data-privacy/enquiry">submit an enquiry</a>. A member of our data privacy team will be in touch with you shortly.</p>
+        <p class="u-sv1">For information about data privacy, please read our <a
+            href="https://www.ubuntu.com/legal/data-privacy">Privacy Policy</a> or <a
+            href="https://ubuntu.com/legal/data-privacy/enquiry">submit an enquiry</a>. A member of our data privacy team
+          will be in touch with you shortly.</p>
         <p class="p-heading--three">Legal enquiries</p>
-        <p>Please note that we cannot provide you with legal advice. For any other queries, please contact us on <a href="mailto:legal@canonical.com">legal@canonical.com</a>.</p>
+        <p>Please note that we cannot provide you with legal advice. For any other queries, please contact us on <a
+            href="mailto:legal@canonical.com">legal@canonical.com</a>.</p>
       </div>
     </div>
   </section>
-{% endblock %}
+  <section class="p-strip--light is-shallow" style="padding-bottom: 6rem;">
+    <div class="u-fixed-width">
+      <p class="p-heading--three">
+        <a href="#" class="p-reveal-link js-location-toggle-button">Where we are</a>
+      </p>
+    </div>
+    <div class="js-location-container">
+      <div class="u-fixed-width">
+        <hr style="margin: 2rem 0;" />
+      </div>
+      <div class="row">
+        <div class="col-4 p-card" style="border-style: none; background-color:#fff;">
+          <p class="p-card__title">Canonical Group Limited</p>
+          <p><small>5th Floor, Blue Fin Building<br>110 Soutwark Street<br>London, SE1 0SU<br>United Kigdom</small></p>
+          <p><a href="https://maps.google.co.uk/maps?q=5th+Floor,+Blue+Fin+Building+110+Southwark+Street+London,+SE1+0SU+United+Kingdom&amp;hl=en&amp;sll=52.8382,-2.327815&amp;sspn=7.78157,20.43457&amp;hnear=Blue+Fin+Bldg,+110+Southwark+St,+London+SE1+0SU,+United+Kingdom&amp;t=m&amp;z=16"><small>View map&nbsp;&rsaquo;</small></a></p>
```

Comment:
Looks like @waynecrosby's comment about making these map links open in a new tab, with a consistent zoom, hasn't been addressed - the anchor tags should all have `target="_blank"`, and it looks like the zoom level can be controlled by the `z` query param i.e. `z=16`, and there's a few different values across the links.